runtime/compiler/runtime/JitRuntime.cpp

```diff
@@ -1522,10 +1522,19 @@ uint8_t *compileMethodHandleThunk(j9object_t methodHandle, j9object_t arg, J9VMT
       TR_VerboseLog::vlogRelease();
       }
    bool disabled = false;
-   if (flags & TRANSLATE_METHODHANDLE_FLAG_CUSTOM)
-      disabled = cmdLineOptions->getOption(TR_DisableCustomMethodHandleThunks);
+#if defined(JITSERVER_SUPPORT)
+   // Do not allow local compilations in JITServer server mode
+   TR::CompilationInfo * compInfo = getCompilationInfo(jitConfig);
+   if (compInfo->getPersistentInfo()->getRemoteCompilationMode() == JITServer::SERVER)
+      disabled = true;
```

Comment:
Could you please insert curly brackets for the `if` here. Our coding convention requires that both parts of the `if` should look the same (with or without curly brackets)